packages/jokul/package.json

```diff
@@ -87,46 +45,45 @@
             "require": "./build/cjs/components/index.js",
             "import": "./build/es/components/index.js"
         },
-        "./styles": "./src/styles/*",
-        "./styles/core/*": "./src/core/styles/*",
-        "./styles/accordion/*": "./src/components/accordion/styles/*",
-        "./styles/breadcrumb/*": "./src/components/breadcrumb/styles/*",
-        "./styles/button/*": "./src/components/button/styles/*",
-        "./styles/card/*": "./src/components/card/styles/*",
-        "./styles/checkbox/*": "./src/components/checkbox/styles/*",
-        "./styles/combobox/*": "./src/components/combobox/styles/*",
-        "./styles/cookie-consent/*": "./src/components/cookie-consent/styles/*",
-        "./styles/datepicker/*": "./src/components/datepicker/styles/*",
-        "./styles/feedback/*": "./src/components/feedback/styles/*",
-        "./styles/description-list/*": "./src/components/description-list/styles/*",
-        "./styles/expander/*": "./src/components/expander/styles/*",
-        "./styles/icon/*": "./src/components/icon/styles/*",
-        "./styles/icon-button/*": "./src/components/icon-button/styles/*",
-        "./styles/input-group/*": "./src/components/input-group/styles/*",
-        "./styles/image/*": "./src/components/image/styles/*",
-        "./styles/link/*": "./src/components/link/styles/*",
-        "./styles/link-list/*": "./src/components/link-list/styles/*",
-        "./styles/list/*": "./src/components/list/styles/*",
-        "./styles/loader/*": "./src/components/loader/styles/*",
-        "./styles/logo/*": "./src/components/logo/styles/*",
-        "./styles/menu/*": "./src/components/menu/styles/*",
-        "./styles/message/*": "./src/components/message/styles/*",
-        "./styles/modal/*": "./src/components/modal/styles/*",
-        "./styles/pagination/*": "./src/components/pagination/styles/*",
-        "./styles/popover/*": "./src/components/popover/styles/*",
-        "./styles/progress-bar/*": "./src/components/progress-bar/styles/*",
-        "./styles/radio-button/*": "./src/components/radio-button/styles/*",
-        "./styles/select/*": "./src/components/select/styles/*",
-        "./styles/summary-table/*": "./src/components/summary-table/styles/*",
-        "./styles/system-message/*": "./src/components/system-message/styles/*",
-        "./styles/table/*": "./src/components/table/styles/*",
-        "./styles/tabs/*": "./src/components/tabs/styles/*",
-        "./styles/tag/*": "./src/components/tag/styles/*",
-        "./styles/text-input/*": "./src/components/text-input/styles/*",
-        "./styles/toast/*": "./src/components/toast/styles/*",
-        "./styles/toggle-switch/*": "./src/components/toggle-switch/styles/*",
-        "./styles/tooltip/*": "./src/components/tooltip/styles/*",
-        "./fonts/*": "./src/fonts/*"
+        "./styles": "./styles/*",
```

Comment:
Kan vi ikke egentlig droppe hele denne blokka også nå? Dt gjør jo ikke noe om man må importere stilen via en path som har "components" i seg og det er vel uansett det som vil dukke opp i auto-completion?

Reply:
Arket med alle stilene ligger jo på rot, så jeg tror vi fortsatt må ha med et entry for det. I hvert fall hvis det skal kunne importeres fra JS-filer.

Reply:
Men den trenger kanskje ikke være stjernet 🤔

Reply:
Hadde i hvert fall vært digg å slippe å huske å legge til en linje for hver komponent hvis vi kan unngå det

Reply:
Skal teste litt med link lokalt

Reply:
Det funka!